resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/DriverCommandFeatureStep.scala

```diff
@@ -73,7 +76,9 @@ private[spark] class DriverCommandFeatureStep(conf: KubernetesDriverConf)
           .withValue(conf.get(PYSPARK_MAJOR_PYTHON_VERSION))
         .build())
 
-    val pythonContainer = baseDriverContainer(pod, res)
+    // re-write primary resource to be the remote one and upload the related file
+    val newResName = KubernetesUtils.renameMainAppResource(res, Some(conf.sparkConf), false)
```

Comment:
We recommend the following stye.
```scala
- val newResName = KubernetesUtils.renameMainAppResource(res, Some(conf.sparkConf), false)
+ val newResName = KubernetesUtils.renameMainAppResource(res, Some(conf.sparkConf), isJava = false)
```
The above is an example. When we rename `isJava`, we should change here together.

Reply:
ok